services/QuillLMS/app/queries/student_dashboard_metrics.rb

```diff
@@ -34,6 +34,7 @@ def completed_sessions
         .joins(:classroom_unit)
         .where(user:, classroom_unit: {classroom_id:})
         .where.not(completed_at: nil)
+        .where(visible: true)
```

Comment:
I think if you just remove `unscoped`, rather than specifying `visible: true`, it'll be automatically scoped to include only visible activities -- I think this is how it works in most places in the codebase now, but I can certainly see an argument that we actually do want to be explicit about this.

Reply:
Initially, I was going to say get rid of the `unscoped` and remove the `where(visible: true)`.  However, on closer inspection, I noticed `ActivitySession` has another `default_scope` for `joins(:activity)`, so I would opt for being explicit here with the `where(visible: true)`.